### DISCUSSION TITLE
Where does filtering take place?

### DISCUSSION THREAD

**Discussion by kdwyer** [eulerto/wal2json#230] [Q&A] [answered]
Hello,

We can filter wal2json's output using the `filter-tables` or `add-tables` parameters.  Does this filtering take place on the server side or the client side?  I'm trying to understand whether filtering will affect the amount of network traffic between the client and the server.

**Comment by eulerto** [comment, marked as answer]
Any output plugin (such as wal2json) has its work done on the provider. In this case, the wal2json will provide messages based on your filter settings and ignore the tables that don't match the filter criteria.
